src/main/java/org/cascadebot/cascadebot/moderation/ModerationManager.java

```diff
@@ -77,6 +77,9 @@ private boolean runChecks(ModAction action, User target, Member submitter, Comma
         } else if (target.equals(context.getSelfUser())) {
             context.getTypedMessaging().replyWarning(context.i18n("moderation_manager.cannot_action_bot", action.getName(context.getLocale())));
             return false;
+        } else if(!submitter.canInteract(context.getGuild().getMember(target)) && context.getData().getModeration().isRespectBanOrKickHierarchy()) {
```

Comment:
`if` keyword and condition needs a space between

Reply:
It'd be good to have this condition the other way round. This is so the condition short-circuits on the less complex method call